docs/src/api/params.md

```diff
@@ -154,6 +154,38 @@ Toggles bypassing page's Content-Security-Policy.
 
 Sets a consistent viewport for each page. Defaults to an 1280x720 viewport. `null` disables the default viewport.
 
+## evaluate-expression
+* langs:
+  - alias-js: pageFunction
```

Comment:
Follow up: we won't need alias here since we have enough info in the override itself.

Reply:
Done.